### PR TITLE
{math}[foss/2018a] PaStiX v5.2.3, PaStiX v6.0.1

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@SimonPinches Please fix the `http` errors.

**Comment by SimonPinches** [comment]
@verdurin I've changed http to https for the homepage URL as requested and have also added variants for complex matrices.

File: easybuild/easyconfigs/p/PaStiX/PaStiX-6.0.1-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://gforge.inria.fr/frs/download.php/file/37632/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['276fff4e95ef0bad1402a7ad40d558722c509e3cc0193fbe509887c639227a30']
+
+dependencies = [
+    ('hwloc', '1.11.8'),
+    ('SCOTCH', '6.0.5'),
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PaStiX/PaStiX-6.0.1-foss-2018a.eb:19]: @SimonPinches oughtn't this to be `6.0.6`? `6.0.5` doesn't exist for that toolchain.

Comment (SimonPinches) [reply easybuild/easyconfigs/p/PaStiX/PaStiX-6.0.1-foss-2018a.eb]: @verdurin This EB file got accidentally caught up in this PR and was not ready for inclusion.  I've therefore now removed it.

**Comment by boegel** [comment]
Closing this, since the `foss/2018a` toolchain is now [deprecated](https://docs.easybuild.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains).

@SimonPinches If you feel this is still relevant, please open a PR with a more recent toolchain

**Pull request closed without merging**